### DISCUSSION TITLE
Cosmetic - SPC Menu Icon

### DISCUSSION THREAD

**Discussion by cheater** [YoshifumiFuyuno/Search-from-Popup-or-ContextMenu#39] [Ideas]
When I was using SSS, its icon kind of annoyed me because it wasn't a "search icon". The SPC icon is much more fitting, and I like it a lot, but I bet there are people out there who would love to change it to their own icon. Therefore please allow changing the icon by the user.

**Comment by YoshifumiFuyuno** [comment]
I might do it, but it's a low priority.

**Comment by cheater** [comment]
I agree.
